doc/api/index.rst

```diff
@@ -63,18 +81,35 @@ Saving and displaying the figure:
     Figure.show
     Figure.psconvert
 
-Configuring the display settings:
+Configuring the display settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following modules are provided directly through the :mod:`pygmt` top level
```

Comment:
There's only one module under here, so grammatically speaking, should be is :slightly_smiling_face:

```suggestion
The following module is provided directly through the :mod:`pygmt` top level
```